csfle/dotnet/aws/reader/CSFLE/YourCredentials.cs

```diff
@@ -11,7 +11,7 @@ class YourCredentials
             {
                 // Mongo Paths + URI
                 {"MONGODB_URI", "<your MongoDB URI>"},
-                {"MONGOCRYPTD_PATH", "<path to mongocryptd>"},
+                {"SHARED_LIB_PATH", "<Full path to your Automatic Encryption Shared Library>"},
```

Comment:
Suggestion:

I think "Absolute path" might be better than "Full" since it's often used in contrast to "relative".
I also think the proper preposition is "of" since "to" seems more related to the action of navigation.

"Absolute path of your ..."